### PR TITLE
Add kubernetes dashboard component

### PR DESCRIPTION
**Pull request by martina-if**
closes https://github.com/weaveworks/aws-workstream/issues/16

### CODE REVIEW

File: cluster-components/kube-system/kubernetes-dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    matchLabels:
+      k8s-app: kubernetes-dashboard
+  template:
+    metadata:
+      labels:
+        k8s-app: kubernetes-dashboard
+    spec:
+      containers:
+      - name: kubernetes-dashboard
+        image: k8s.gcr.io/kubernetes-dashboard-amd64:v1.10.1
```
Comment (stefanprodan) [inline cluster-components/kube-system/kubernetes-dashboard.yaml]: Shouldn't we use the v2 release that supports the metrics-server? v1 only works with [heapster](https://github.com/kubernetes-retired/heapster) but that's deprecated. If we want the v1 dashboard then we should deploy heapster as well (heapster will not work on newer Kubernetes cluster).

Comment (stefanprodan) [reply cluster-components/kube-system/kubernetes-dashboard.yaml]: Here is the YAML for v2 https://raw.githubusercontent.com/kubernetes/dashboard/v2.0.0-beta2/aio/deploy/recommended.yaml

Comment (martina-if) [reply cluster-components/kube-system/kubernetes-dashboard.yaml]: Thanks!

**Comment by martina-if** [comment]
What do you think about the folder layout?

**Comment by cPu1** [comment]
Based on our discussion of cloning the base repo (`eks-gitops-example`) and generating overlays, the content of this repo is meant to go inside a `base` directory under the user's git repo:

```
├── base (upstream git repo containing base configuration cloned locally)
│   ├── kube-system
│   │   ├── aws-alb-ingress-controller
│   │   │   ├── deployment.yaml
│   │   │   └── rbac-role.yaml
│   │   ├── tiller-cluster-role-binding.yaml
│   │   ├── tiller-deploy-dep.yaml
│   │   ├── tiller-deploy-svc.yaml
│   │   └── tiller-service-account.yaml
│   ├── kustomization.yaml
│   ├── monitoring
│   │   ├── 0-namespace.yaml
│   │   ├── metrics-server.yaml
│   │   └── prometheus.yaml
│   └── system
│       ├── flux-helm-release-crd.yaml
│       └── helm-operator-deployment.yaml
├── overlays (generated by `eksctl install gitops` using cluster-specific information)
│   └── cluster-components
│       ├── alb-patch.yaml
│       ├── grafana-patch.yaml
│       ├── kustomization.yaml
│       └── prometheus-rules-patch.yaml
└── README.md
```

To achieve this structure, the directory `cluster-components` in this PR should be removed to make `kube-system` (and other namespaces) the top-level directory. Although I can see that having these manifest directories at the top level may result in inconsistency if we were to add files other than Kubernetes and Helm manifests later.

**Review by cPu1 — APPROVED** [review]

**Pull request merged**